src/pages/Sketch/components/Canvas.jsx

```diff
@@ -461,14 +464,16 @@ function Canvas() {
       accepts: [
         {
           description: "Doc2pen Sketch Save File",
-          extensions: ["d2ps"],
-          mimeTypes: ["application/d2ps"],
+          extensions: [".png"],
+          mimeTypes: ["image/*"],
```

Comment:
`getNewFileHandle` funciton is edited. This function is meant to **"SAVE"**  d2ps files. It is not meant to **LOAD** or **IMPORT** image files.

Thus, completely obsolete code is added here blindly.

Because of this change, d2ps files are not save with correct mimeType

Reply:
i know buddy!! but for making it more suitable i changed the file format for saving the progress so that we can view it easily.

Reply:
> but for making it more suitable i changed the file format for saving the progress 

progress is saved in a special file format ".d2ps". this format is not an image. Its a raw buffer. not an image
Lets not argue upon this. Nobody is putting blame.
I reviewed this code in this PR in order to point out the flaws only.